app/lib/request_utils.rb

```diff
@@ -82,6 +82,7 @@ def self.sanitize_search_terms(terms)
   # takes a comma-delimited string of ids (e.g. StudyFile ids) and returns an array of ids
   # raises Argument error if any of the strings are not valid ids
   def self.validate_id_list(id_list_string)
+    byebug
```

Comment:
```suggestion
   
```

Reply:
Great catch.  

Flagging active `byebug` calls would be a high-value lint rule.  It's easy to automate, often missed by humans, and will crash at runtime.

Reply:
This is enabled by default in `rubocop` once we actually turn it on...